src/bluetooth/TemperatureLogManager.js

```diff
@@ -74,3 +75,14 @@ export class TemperatureLogManager {
 
   saveLogs = async logsToSave => this.db.upsertTemperatureLog(logsToSave);
 }
+
+let TemperatureLogManagerInstance;
+
+export const getTemperatureLogManagerInstance = (dbService, utils) => {
```

Comment:
I'm not sure about this - would it be better to expose a setter similar to what we've done for BleService?

Reply:
Good like this, I think.